src/components/Modal/ModalContent/ModalContent.tsx

```diff
@@ -0,0 +1,79 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import * as React from 'react';
+import { TestId } from '../../../utils/types';
+import {
+  actionsContainer,
+  headingContainer,
+  labelContainer,
+  messageContainer,
+  modalContentContainer,
+} from './ModalContent.style';
+import { generateTestDataId } from '../../../utils/helpers';
+import Button from '../../Button';
+
+export type Props = {
+  /** The label of the modal. */
+  label?: string;
+  /** The heading of the modal. */
+  heading: string;
+  /** The main messager of the modal. */
+  message: string;
+  /** The primary call-to-action label of the Modal */
+  primaryCTALabel?: string;
+  /** The primary call-to-action of the Modal */
+  primaryCTA?: () => void;
+  /** The secondary call-to-action label of the Modal */
+  secondaryCTALabel?: string;
+  /** The secondary call-to-action of the Modal */
+  secondaryCTA?: () => void;
+  /** The data test id if needed */
+  dataTestId?: TestId;
+};
+
+const ModalContent: React.FC<Props> = ({
+  label,
+  heading,
+  message,
+  primaryCTA,
+  primaryCTALabel,
+  secondaryCTA,
+  secondaryCTALabel,
+  dataTestId,
+}) => {
+  return (
+    <div
```

Comment:
You can change this to `dialog` to make it semantically correct.

Reply:
Don't. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog#browser_compatibility

Reply:
`role="dialog"` then 😉